### PR TITLE
Fix timezone related issues in calendar

### PR DESCRIPTION
**Pull request by pandafulmanda**
Updates various time dependencies and fixes timezone issue in calendar

For [timezone bug ticket](https://www.pivotaltracker.com/story/show/117736371)

### CODE REVIEW

File: src/components/task-plan/builder.cjsx [hunk 1]
```diff
@@ -377,9 +369,10 @@ module.exports = React.createClass
 
   renderEnabledTasking: (plan) ->
     {taskingOpensAt, taskingDueAt} = @getDefaultPlanDates(plan.id)
-    maxOpensAt = new moment(TaskPlanStore.getDueAt(@props.id, plan.id)).subtract(1, 'day').toDate()
+    maxOpensAt = TimeHelper.makeMoment(TaskPlanStore.getDueAt(@props.id, plan.id), ISO_DATE_FORMAT).subtract(1, 'day')
+    minDueAt = TimeHelper.makeMoment(TaskPlanStore.getMinDueAt(@props.id, plan.id), ISO_DATE_FORMAT)
 
-    <BS.Row key={plan.id} className="tasking-plan tutor-date-input">
+    <BS.Row key="tasking-enabled-#{plan.id}" className="tasking-plan tutor-date-input">
```
Comment (philschatz) [inline src/components/task-plan/builder.cjsx]: what does this key conflict with?

**Comment by philschatz** [comment]
Can you add Selenium helpers and a test so @openstax/test has a way to target and edit the timezone?

File: src/components/task-plan/builder.cjsx (hunk 1 above)
Comment (pandafulmanda) [inline src/components/task-plan/builder.cjsx]: This was just to make it more clear, I thought there was a bug due to this, but there wasn't.

**Comment by philschatz** [comment]
ah, #1058 adds the setting to edit the timezone!

**Pull request merged**